config/final.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Yes, it does, it was on purpose because we want to use the same package than cf-release.